frontend/packages/dev-console/src/components/edit-deployment/EditDeploymentForm.tsx

```diff
@@ -82,7 +83,7 @@ const EditDeploymentForm: React.FC<FormikProps<FormikValues> & {
   }, [editorType, resource, setErrors, setFieldValue, setStatus, formReloadCount]);
 
   return (
-    <FlexForm onSubmit={handleSubmit}>
+    <FlexForm onSubmit={handleSubmit} style={{ minHeight: '80px' }}>
```

Comment:
@Lucifergene what was the need to add `minHeight`?

Reply:
Actually, there was a CSS issue that caused a double scrollbar. This is not reproducible every time. This CSS fix was for that, but still, it did not solve the issue.
I am removing it then.